proxy-node-shared/src/main/java/uk/gov/ida/notification/contracts/EidasSamlParserRequest.java

```diff
@@ -7,9 +7,6 @@
     @JsonProperty
     private String authnRequest;
 
-    public EidasSamlParserRequest() {
```

Comment:
I think the default constructor is needed for Jackson to serialise the DTO.

Reply:
I think you're right!